native/src/main/java/tech/sourced/babelfish/TranslationUnitJSONMapper.java

```diff
@@ -10,20 +10,21 @@
 import com.fasterxml.jackson.databind.module.SimpleModule;
 import org.eclipse.cdt.core.dom.ast.IASTTranslationUnit;
 
-import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
 import java.io.IOException;
 
 class TranslationUnitJSONMapper implements IExchangeFormatWritter {
 
     final JsonGenerator generator;
     final JsonFactory jsonFactory = new JsonFactory();
     final ObjectMapper mapper = new ObjectMapper();
-    private ByteArrayOutputStream byteOutputStream;
+    private OutputStream printStream;
 
-    TranslationUnitJSONMapper(boolean prettyPrint, ByteArrayOutputStream byteOutput) throws IOException {
-        this.byteOutputStream = byteOutput;
+    TranslationUnitJSONMapper(boolean prettyPrint, PrintStream byteOutput) throws IOException {
```

Comment:
Nit for future consideration: It's no longer `byteOutput`.

Reply:
As above (but can change it).

Reply:
It's not urgent, it just stuck out because usually the convention is to match the name to the type.